locales/sv.lua

```diff
@@ -1,19 +1,19 @@
 local Translations = {
     notify = {
-        ["hud_settings_loaded"] = "HUD-inställningar laddade!",
-        ["hud_restart"] = "HUD startas om!",
+        ["hud_settings_loaded"] = "HUD Inställningar laddade!",
```

Comment:
Why removing the dash? It's supposed to be there according to how swedish language works